.github/scripts/install-slices/install_slices.py

```diff
@@ -280,6 +294,44 @@ def install_slice(slice: str, arch: str, release: str) -> None:
                 res.stderr.rstrip(),
             )
             sys.exit(res.returncode)
+        # Check if the copyright file has been installed with this slice
+        copyright_file = pathlib.Path(f"{tmpfs}/usr/share/doc/{pkg}/copyright")
+        installed_copyright[slice_name] = (
+            copyright_file.is_file() or copyright_file.is_symlink()
+        )
+
+
+def deb_has_copyright_file(pkg: str) -> bool:
+    """
+    Checks if a deb's contents comprise a copyright file
+
+    NOTE: this is a temporary and convoluted implementation, as at the moment
+    we don't have an easy and reliable way to check which deb was used for
+    the installation (at least not without duplicating a some of the Chisel
+    codebase).
+
+    TODO: update this function once the Chisel DB is available, as the pkg
+    SHAs will be available from the DB itself.
+    """
+    for sha_file in pathlib.Path(CHISEL_PKG_CACHE).rglob("*"):
+        try:
+            sha_type = magic.from_file(str(sha_file), mime=True)
+        except:
+            # Ignore any other kind
+            continue
+
+        if sha_type and "debian.binary-package" in sha_type:
+            deb_path = str(pathlib.Path(CHISEL_PKG_CACHE / sha_file))
+            sha_pkg = os.popen(
+                f"dpkg-deb -f {deb_path} Package"
+            ).read().strip()
+
+            if sha_pkg == pkg:
+                deb = DebPackage(deb_path)
+                if f"usr/share/doc/{pkg}/copyright" in deb.filelist:
+                    return True
```

Comment:
[Nit]: `return f"usr/share/doc/{pkg}/copyright" in deb.filelist`.

Reply:
see 0bf534c